float32 lightInt
bool sensor_attached